src/storage/browser.ts

```diff
@@ -57,7 +57,8 @@ export class BrowserStorage implements IDriveStorage {
         drive: string,
         id: string,
         operations: Operation[],
-        header: DocumentHeader
+        header: DocumentHeader,
+        updatedOperations: Operation[] = []
```

Comment:
`addDocumentOperations` now receives a new argument called `updatedOperations` that contains the operations updated.

`updatedOperations` are not included into `operations` argument